demo/ksnav_app_linux/ksnav_start.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
```

Comment:
Move steps into a code block when called in readme

Reply:
Integrated in latest suite of commits